portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,29 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Kayla is 19 years old'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function KaylaTrivia(){
```

Comment:
This can be done outside the scope of this PR, but if you ever have time to come back to this, consider working on some of these stretch goals: 

- Consider what other data structures may be helpful here. In this case we have both `questions` and `correct_answers` which are of the same length and intended to be used together. 
- Break this function up to use some of its parts as helper functions. For example, I can imagine one helper function which takes a question and answer and returns a boolean for correctness. Another one I can imagine is one function which takes a set of questions and administers the quiz with alerts. Another one still might take a final score and respond accordingly. There are other ways you might consider breaking this up but there are a few ideas.
- Finally, practice writing more concise and informative JS doc function comments. The JS style guide discusses this more in depth: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
I'll come back to add comments and break this up. I didn't think about that initially :(